fixity/utils.py

```diff
@@ -33,3 +34,10 @@ def check_valid_uuid(uuid):
 
 def utcnow():
     return datetime.now(timezone.utc)
+
+
+def pyprint(message):
+    if message.startswith("Internal error encountered"):
```

Comment:
Why does this depend on the contents of the `message`? Suppose there's new code that adds another message that starts with the same substring but needs to send the output to standard error instead.

Could this be implemented with a keyword argument like the [`Job.pyprint` method](https://github.com/artefactual/archivematica/blob/bba53bad33c0ce0a6453fedc3f5b62bda303faa3/src/MCPClient/lib/client/job.py#L111-L122) does in the MCPClient?